setup.py

```diff
@@ -584,7 +584,7 @@ def run(self):
 
     def patch_version(self, filename):
         regex = re.compile(r'^PICARD_BUILD_VERSION_STR\s*=.*$', re.MULTILINE)
-        with open(filename, 'r+b') as f:
```

Comment:
Doing so will possibly change end of lines of the read then written file. Instead if  i think we should keep binary mode, but convert source to unicode.